docs/Cakefile.md

```diff
@@ -300,15 +300,32 @@ end
 
 ##### Shell Script Build Phase
 
-You can create a Shell Script buld phase to run a script when building.
+You can create a Shell Script build phase to run a script when building.
+The following creates a simple script printing `Hello World`:
 
 ```ruby
-target.shell_script_build_phase "Build Phase Name", <<-SCRIPT 
-  echo "Hello World"
+target.shell_script_build_phase "Build Phase Name", "echo 'Hello World'"
+end
+```
+
+You implement multi-line scripts, and define input- and output (file list) paths like this:
+
+```ruby
+myScript = <<-SCRIPT
+    echo "This is a multi-line script"
+    echo "Hello World"
 SCRIPT
+target.shell_script_build_phase "Build Phase Name", myScript do |phase|
+    phase.input_paths = ["$(SRCROOT)/$(TARGET_NAME)/**/*.txt"]
+    phase.output_paths = ["$(SRCROOT)/$(TARGET_NAME)/OutputFiles/MyFile.txt"]
+    phase.input_file_list_paths = []
```

Comment:
add an example of a file list to input & output like `["$(SRCROOT)/$(TARGET_NAME)/OutputFiles/MyFileList.xcfilelist"]`

Reply:
done